content/blog/ggplot2-3-5-0/index.Rmd

```diff
@@ -0,0 +1,385 @@
+no_expand <- pie + coord_radial(expand = FALSE) + ggtitle("expand = FALSE")
+polar     <- pie + coord_polar() + ggtitle("coord_polar()")
+
+default | no_expand | polar
+```
+
+Some visual differences stand out in the plots above.
+In `coord_radial()`, the panel background covers the data area of the plot, not a rectangle.
+It also does not have a grid-line encircling the plot and instead uses tick marks to indicate values along the theta (angle) coordinate.
+
+Another important difference is that `coord_radial()` does not necessarily display a full circle.
+By setting the `start` and `end` arguments separately, you can make a partial polar plot.
+One benefit is that the radius axis can now be placed closer to the data, and doesn't have to awkwardly dangle at the edge of the panel.
+Additionally, one can avoid cramming data in the center of the plot by using the `inner.radius` argument.
+
+```{r open_radial_plot}
+#| fig.alt = "A horseshoe-shaped plot in polar coordinates with an open top
+#|  and a hole in the middle."
+pie + coord_radial(start = 0.25 * pi, end = 1.75 * pi, inner.radius = 0.3)
+```
+
+Another drawback about polar coordinates is that it can become pretty involved to coordinate the angles of text annotations with the theta (angle) position of the plot.
+To remove some of this hassle, `coord_radial()` has a `rotate_angle` switch, that will line up the text's `angle` aesthetic with the theta coordinate.
+For text angles of 0 degrees, this will place text in a tangent orientation to the circle and for angles of 90 degrees, this places text along the radius, as in the plot below.
+
+```{r text_angles}
+#| fig.alt = "A wind rose plot showing miles per gallon for different cars. The
+#|  car names skirt the outer edge of the plot and are oriented towards the
+#|  centre."
+ggplot(mtcars, aes(seq_along(mpg), mpg)) +
+  geom_col(width = 1) +
+  geom_text(
+    aes(y = 32, label = rownames(mtcars)),
+    angle = 90, hjust = 1
+  ) +
+  coord_radial(rotate_angle = TRUE, expand = FALSE)
+```
+
+Because the logic of drawing the axes for polar coordinates is not the same as when the axes are vertical or horizontal, we used the new guide system to build an axis specific to `coord_radial()`: the `guide_axis_theta()` axis.
+Guides for `coord_radial()` can be set using the `theta` and `r` name in the `guides()` function, where the `r` guide can be a regular axis.
+The theta guide obeys the theme settings of `axis.{setting}.x.bottom` to display the axis, regardless of whether `theta = "x"` or `theta = "y"` in `coord_radial()`. The primary r-axis likewise uses the `axis.{setting}.y` setting.
+When setting the `angle` parameter in the guide function, these are considered 'relative' to the coordinates.
+Setting `angle = 0` will align the `r`-axis text with the tick marks and the `theta`-axis following the tangent.
+
+```{r axis_styling}
+#| fig.alt = "A pacman-shaped plot with an open quadrant at the top. An axis
```

Comment:
I'd be somewhat tempted to pull out the whole section about radial charts, combine it with the radial axes from the other post, and make it it's own post.